yb-voyager/cmd/callhome_payload_builder.go

```diff
@@ -61,9 +61,10 @@ func packAndSendAssessMigrationPayload(status string, errMsg error) {
 	payload.Status = status
 	if assessmentMetadataDirFlag == "" {
 		sourceDBDetails := callhome.SourceDBDetails{
-			DBType:    source.DBType,
-			DBVersion: source.DBVersion,
-			DBSize:    source.DBSize,
+			DBType:                   source.DBType,
+			DBVersion:                source.DBVersion,
+			DBSize:                   source.DBSize,
+			PostgresSystemIdentifier: source.PostgresSystemIdentifier,
```

Comment:
I would recommend not using "Postgres" in the key/field name. This should be source-agnostic, and re-usable in the future.

Reply:
Hmm I was confused about this. But yeah I'll make it source agnostic